guides/source/active_record_basics.md

```diff
@@ -21,27 +21,27 @@ What is Active Record?
 ----------------------
 
 Active Record is the M in [MVC][] - the model - which is the layer of the system
-responsible for representing business data and logic. Active Record facilitates
-the creation and use of business objects whose data requires persistent storage
-to a database. It is an implementation of the Active Record pattern which itself
-is a description of an Object Relational Mapping system.
+responsible for representing data and business logic. Active Record helps you
+create and use Ruby objects whose attributes require persistent storage
+to a database.
+
+Note that it's also possible to model data with Ruby objects that do *not* need to be backed by a database. [Active Model](active_model_basics.html) is used for that. So Active Record and Active Model together are the M in MVC.
+
+There is some industry jargon around Active Record. Active Record in Rails is an implementation of a software architecture pattern by the same name. It's also a description of something called [Object Relational Mapping][ORM] system. The below sections explain these terms:
```

Comment:
Just trying to differentiate the two "Active Record" mentions that are read together:

```suggestion
There is some industry jargon around the term "Active Record". Active Record in Rails is an implementation of a software architecture pattern by the same name. It's also a description of something called an [Object Relational Mapping][ORM] system. The below sections explain these terms:
```